mne/viz/topo.py

```diff
@@ -184,7 +184,7 @@ def _plot_topo(info, times, show_func, click_func=None, layout=None,
         layout.pos[:, :2] /= layout.pos[:, :2].max(0)
 
     # prepare callbacks
-    tmin, tmax = times[[0, -1]]
+    tmin, tmax = times[0], times[-1]
```

Comment:
what is the difference here? Why not `[[0, -1]]`?

Reply:
I assumed a stylistic symmetry choice ... ?

Reply:
Iirc times is no longer a numpy array, so it cannot be indexed by lists anymore.

Reply:
In fact the call signature is (and was) a bit weird, since that function always just picked tmin and tmax from times, and could be called with tmin and tmax as separate arguments. I was too lazy to fix this, but I can do it if you guys insist.